var/spack/repos/builtin/packages/mvdtool/package.py

```diff
@@ -22,33 +22,23 @@ class Mvdtool(CMakePackage):
     version('1.5', tag='v1.5')
     version('1.4', tag='v1.4')
 
-    variant('mpi', default=True, description="Enable MPI backend")
     variant('python', default=False, description="Enable Python bindings")
 
     depends_on('boost')
     depends_on('cmake', type='build')
     depends_on('py-setuptools-scm', type='build', when='@2:')
 
-    depends_on('hdf5+mpi', when='+mpi')
-    depends_on('hdf5~mpi', when='~mpi')
-    depends_on('highfive+mpi', when='+mpi')
-    depends_on('highfive~mpi', when='~mpi')
-    depends_on('mpi', when='+mpi')
+    depends_on('hdf5')
+    depends_on('highfive~mpi')
```

Comment:
The problem is that mixing ~mpi with +mpi is an error, right? Maybe we should have something like Neuron: dont link with mpi but dlopen() and mpi_init when the user requests?

Reply:
I should actually close this. With `spack` we'll run into all sorts of concretizing issues :(